pyproject.toml

```diff
@@ -35,7 +35,7 @@ psutil = "^5.9"
 appdirs = "^1.4"
 click = "^7.0"  # requires bump to goth's dependencies https://github.com/golemfactory/goth/issues/605
 pydantic = "^1.9"
-dapp-runner = { git = "https://github.com/golemfactory/dapp-runner.git", branch = "main" }
+dapp-runner = { git = "https://github.com/golemfactory/dapp-runner.git", branch = "blue/suspend-resume" }
```

Comment:
Is this to be merged?

Reply:
no, but until the dapp-runner change is merged, this feature won't work on main